source/github-helpers/github-file-url.ts

```diff
@@ -80,7 +80,7 @@ export default class GitHubFileURL {
 				repository,
 				route,
 				branch,
-				filePath: '',
+				filePath: ambiguousReference[1].replaceAll('%2F', '/'),
```

Comment:
The function has worked for years so it's unlikely that this is the wrong part.

Reply:
In fact, we're in the `isRepoRoot` condition. The path must be `''`

Given the TODO above the condition, it's likely that the function must be updated in `github-url-detection` instead